common/service/ipmb/ipmb.c

```diff
@@ -46,7 +46,7 @@ LOG_MODULE_REGISTER(ipmb);
 #if MAX_IPMB_IDX
 
 static struct k_mutex mutex_id[MAX_IPMB_IDX]; // mutex for sequence linked list insert/find
-static struct k_mutex mutex_send_req[MAX_IPMB_IDX], mutex_send_res, mutex_read;
+static struct k_mutex mutex_send_req[MAX_IPMB_IDX], mutex_send_res, mutex_read, mutex_put_msgq;
```

Comment:
@DelphineChiu Rename this mutex to mutex_purge_msgq since it creates critical section for get msgq size and purge msgq?